### DISCUSSION TITLE
Trouble in compilation during install process

### DISCUSSION THREAD

**Discussion by krit504** [ChampSim/ChampSim#478] [Q&A]
Im following all the procedures for installing Champsim and im currently on this part
$ ./config.sh <configuration file>
$ make
during the make process im returned with an error 1.
Please help me as to what could be the source of this error

**Comment by GinoAC** [comment]
Hi! You need to provide the config.sh script with a json configuration file before running make, for example:
./config.sh champsim_config.json

**Reply by ngober** [reply]
The configuration step will provide a default if no file is given. I think this is unlikely to be the source of the error.

**Reply by GinoAC** [reply]
Ah fair enough.

**Comment by GinoAC** [comment]
Follow-up, what OS are you trying to compile on?
